client/tests/e2e/specs/test_folders_page.ts

```diff
@@ -130,6 +150,20 @@ describe('Check folders page', () => {
     cy.get('.question-modal').find('#next-button').click();
     cy.get('.question-modal').should('not.exist');
     cy.get('@consoleLog').should('have.been.called.with', /File File_[a-z_]+ deleted/);
+
+    // grid view
+    cy.get('#folders-ms-action-bar').find('#grid-view').as('gridButton');
```

Comment:
Move it to another test.

Reply:
Done

Reply:
Naaah. I just meant add another test in the same file to test deletion in grid mode. Something like

```typescript
it('Test delete one file list mode', () => {

});

it('Test delete on file grid mode', () => {

});
```